.github/workflows/release.yml

```
@@ -0,0 +1,42 @@
+name: "Build and push Docker Image to GPASLTD OCIR"
+run-name: "${{ github.repository }} release build"
+concurrency: "${{ github.repository }}-release"
+
+on:
+  release:
+    types: [ published ]
+
+env:
+  REGISTRY: "lhr.ocir.io/lr3yhdniv6gu"
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository ${{ github.repository }}
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Extract repository name
+        id: repo-name
+        # Strip the owner and `/` from the repo env var
+        run: echo "REPOSITORY_NAME=${GITHUB_REPOSITORY#*/})" >> $GITHUB_ENV
```

```suggestion
        run: echo "REPOSITORY_NAME=${GITHUB_REPOSITORY#*/}" >> $GITHUB_ENV
```
It seems there is a redundant character here